### PR TITLE
First attempt to address string issues from localization team

### PR DESCRIPTION
**Pull request by devinreams**
Connected to https://github.com/mozilla-l10n/lockwiseios-l10n/issues/4

### CODE REVIEW

File: CredentialProvider/Storyboard/Base.lproj/CredentialItemList.storyboard [hunk 1]
```diff
@@ -172,7 +172,7 @@
                                                     <constraints>
                                                         <constraint firstAttribute="width" constant="280" id="WcY-qh-pBg"/>
                                                     </constraints>
-                                                    <string key="text">Firefox Lockbox lets you access passwords you've already saved to Firefox. To view your entries here, you'll need to sign in and sync with Firefox for desktop.</string>
+                                                    <string key="text">Firefox Lockbox lets you access passwords youve already saved to Firefox. To view your entries here, you’ll need to sign in and sync with Firefox for desktop.</string>
```
Comment (Pike) [inline CredentialProvider/Storyboard/Base.lproj/CredentialItemList.storyboard]: s/youve/you’ve/

File: Shared/Common/Resources/BaseConstants.swift [hunk 2]
```diff
@@ -58,18 +58,18 @@ class Constant {
...
+        static let completedEnablingAutofill = NSLocalizedString("autofill.finished_enabling", value: "Finished updating AutoFill", comment: "Accesibility notification when AutoFill is done being enabled")
+        static let unlockPlaceholder = NSLocalizedString("unlock_placeholder", value: "This will unlock the app.", comment: "Placeholder text when the user’s email is unavailable while unlocking Lockbox, shown in Touch ID and passcode prompts")
+        static let signInRequired = NSLocalizedString("autofill.signInRequired", value: "Sign in Required", comment: "Title for alert dialog explaining that a user must be signed in to use AutoFill.")
+        static let signInRequiredBody = NSLocalizedString("autofill.signInRequiredBody", value: "You must be signed in to %@ before AutoFill will allow you to add passwords from %@. Once you have signed in, your entries will start appearing in AutoFill.", comment: "Body for alert dialog explaining that a user must be signed in to use AutoFill. AutoFill should be localized to match the proper name for Apple's system feature")
```
Comment (Pike) [inline Shared/Common/Resources/BaseConstants.swift]: Can you add a comment on what %@ will be replaced with? Brand I guess, for both?

Note, in the xliff file, they're `%1$@` and `%2$@`.

File: Shared/Common/Resources/BaseConstants.swift [hunk 3]
```diff
@@ -58,18 +58,18 @@ class Constant {
...
-        static let usernamePlaceholder = NSLocalizedString("username_placeholder", value: "(no username)", comment: "Placeholder text when there is no username")
+        static let usernamePlaceholder = NSLocalizedString("username_placeholder", value: "(no username)", comment: "Placeholder text when there is no username. String should include appropriate open/close parenthetical or similar symbols to indicate this is a placeholder, not a real username.")
         static let searchYourEntries = NSLocalizedString("search.placeholder", value: "Search your entries", comment: "Placeholder text for search field")
-        static let emptyListPlaceholder = NSLocalizedString("list.empty", value: "%@ lets you access passwords you've already saved to Firefox. To view your entries here, you'll need to sign in and sync with Firefox for desktop.", comment: "Label shown when there are no logins to list")
+        static let emptyListPlaceholder = NSLocalizedString("list.empty", value: "%@ lets you access passwords you’ve already saved to Firefox. To view your entries here, you’ll need to sign in and sync with Firefox for desktop.", comment: "Label shown when there are no logins to list")
```
Comment (Pike) [inline Shared/Common/Resources/BaseConstants.swift]: Also, add to the comment what %@ is gonna be?

**Comment by devinreams** [comment]
Thank you, @Pike. I'm sorry I overlooked adding some obvious interpolation comments. I appreciate your thorough review. 

I've addressed your notes here and will merge as-is then continue more rebrand string updates and fixes while we "swarm" today over at https://github.com/mozilla-lockwise/lockwise-ios/tree/921-rebrand

**Pull request merged**